src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
 public class Application {
+    private static final int NUMBER_OF_BALLS = 3;
+
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        for (int i = 0; i < NUMBER_OF_BALLS; i++) {
+            System.out.println(new Lottery().getRandomBall().toString());
+        }
```

Comment:
Create a variable for Lottery class instead of creating it every time. 
Don't call method `toString()`. `System.out.println` calls it by default 
```suggestion
        Lottery lottery = new Lottery();
        for (int i = 0; i < NUMBER_OF_BALLS; i++) {
            System.out.println(lottery.getRandomBall());
        }
```